todo_CLI.js

```diff
@@ -0,0 +1,142 @@
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const fs = require('fs');
+
+function readTodoList(){
+  //expect an array return
+  if(fs.existsSync('todo.json') === true){
+    try {
+      let fileData = fs.readFileSync('todo.json');
+      let parsedFileData = JSON.parse(fileData);
+      if (Array.isArray(parsedFileData)) {
+        return parsedFileData;
+      }
+    } catch (e) {
+     console.log(e);
+    }
+  }
+  return [];
+}
+
+function writeOrCreateFile(fileData) {
+  //if there is no todojson file, create one
+  if (fs.existsSync('todo.json') === false) {
+    let createStream = fs.createWriteStream('todo.json');
+    createStream.end();
+  }
+  fs.writeFile('todo.json', JSON.stringify(fileData), function (err) {
+    if (err) throw err;
+    todoStart();
+  });
+}
+
+function todoStart() {
+  readline.question(
+    '(v) View • (n) New • (cX) Complete • (dX) Delete • (q) Quit ',
+    (input) => {
+      handleAction(input);
+    }
+  )
+}
+
+function fallback() {
+  console.log("We don't have this option, try again")
+  todoStart();
+}
+
+function handleView() {
+  let todolistData = readTodoList();
```

Comment:
do not use `let` unless you are changing the value. make sure you understand the difference between `let` vs. `const`